# ISCB2017

## Draft Overview of the week

## Idea: Little modules (theory.. and respective hands-on session)

**Listing of modules (buzzword & idea for hands-on & Duration & repsonisble person & what still needs to be done):**



## Day 1 (Common Day)

- Morning Session: Responsible SE -> Ingrid & Urs
- Afternoon Session: Responsible PrXIT -> Jonas




## Day 2 (Protein Identifications)

Module Number | Theoretical Part | Hands-on Module | Duration | Main responsible Person
------------ |------------ | ------------- | ------------- | ------------- 
1 | Outline for this week | break | 0.5 h | jg 
2 | The FGCZ in Zurich | browse FGCZ page | 0.5 h | jg
3 | Mass spectrometry (and protein analytics) | browse around for milestones in MS, form groups, introduce yourself | 1 h | jg 
4 | Getting ready for hands-on sessions | Install MQ, do git co, commits, get R packages | 1 h | CP
5 | [PSM identification with protViz](vignettes/peakplot.Rmd) | get fasta, R (seqinR), R (digest), theoretical considerations about search space, different DBs, uniqueness of peptides.  | 2h | CP 
6 | Protein identification VALIDATION with target-decoy strategy | [FDR](vignettes/fdr.Rmd)  | 2h | WEW 
7 | Start your own MQ | run it on your PC | 1h | jg
8 | Conclusion of day 1 | Q & A | 1h | all



## Day 3 (Protein Quantification)

Module Number | Theoretical Part | Hands-on Module | Duration | Main responsible Person
------------ |------------ | ------------- | ------------- | ------------- 
1 | Outline for this week | break | 0.5 h | jg 












————————————————————
Preparations
————————————————————




### Day 1

KeepOrSkip | Theory Module | Hands-on Module | Duration | Responsible person | Action needed
------------ |------------ | ------------- | ------------- | ------------- | ------------- 
Keep | Outline for this week | break | 0.5 h | jg | tbd at the very end (show 1)
Keep | The FGCZ in Zurich | browse FGCZ page | 0.5 h | jg | Jonas (show 2) -> DONE
Keep | Mass spectrometry (and protein analytics) | browse around for milestones in MS, form groups, introduce yourself | 1 h | jg | Combo course day2 first slides (show 3)  -> DONE
Keep | Getting ready for hands-on sessions | Install MQ, do git co, commits, get R packages | 2 h | CP | to-be-prepared (show 4)
Keep | [PSM identification with protViz](vignettes/peakplot.Rmd) | get fasta, R (seqinR), R (digest), theoretical considerations about search space, different DBs, uniqueness of peptides.  | 2h | CP | workshop to be prepared (cp) (show 5)
Keep | Protein identification with search engines / include [FDR](vignettes/fdr.Rmd) | tbd  | 2h | WEW | workshop to be prepared (cp) (show 5)
Keep | [MS and different activiation techniques](vignettes/peakplot.Rmd) | protViz & mgfs & labelling of spectra | 1h | CP?? | (show 6)
~~Skip~~ | ~~PTMs~~ | ~~restricted and unrestricted search, spectral library search (wew)~~ | ~~1h~~ | ~~WEW??~~ | ~~Theo part (ETH lecture) , workshop! (show 7)~~
Keep | Experimental design in proteomcis experiments | Annotation of samples in Bfabric | 1h | jg | to-be-prepared (show 8)
Keep | Conclusion of day 1 | Q & A | 1h | xxx | to-be-prepared (show 9)

### Day 2

KeepOrSkip | Theory Module | Hands-on Module | Duration | Responsible person | Action needed
------------ |------------ | ------------- | ------------- | ------------- | -------------  
Keep | Repetition and wrap up of Day 1 | Q & A | 0.5 h | jg | to-be-prepared (conclusion day1) (show 10)
Keep | Tools for protein identification | Look at MQ and start a search | 1h | jg | to-be-prepared (show 11)
PrepareButSkip | Quantitative proteomics strategies (an overview) | break | 1h | jg | Combo-course day4 (NSK) (show 12)
Keep | Label-free quantification | MQ: txt-tables | 1h | jg | Combo-course day4 (partI) (show 13) 
Keep | Quantitative protein matrices | Visualizing QMs in R. R-quantable  | 1h | (wew?) | Combo-course day4 (partII) (show 14)
Keep | Two group analysis for Yeast, grown on different nutrient sources | Analysing 2 groups in R-quantable | 2h | (wew?) | to-be-prepared (show 15)
Keep | Principle of ORAs |  Webgestalt w YEAST results | 1h | jg | Combo-course day4 (partI) (show 20)
Keep | Webtools for model organisms | Visualizing results on a cellular overview map (YeastCyc) | 1h | cpNjg | to-be-prepared (show 19)
Keep | Conclusion of day 2 | Q & A | 1h | jg | to-be-prepared (show 16)



### Day 3
KeepOrSkip |Theory Module | Hands-on Module | Duration | Responsible person | Action needed
------------ |------------ | ------------- | ------------- | ------------- | -------------  
~~Skip~~ | ~~Repetition and wrap up of Day 2~~| ~~Q & A ~~| ~~0.5 h~~ | ~~cp~~ | ~~to-be-prepared (show 17)~~
~~Skip~~ | ~~Model organisms vs custom databases / blast ~~| ~~Pigeon pea DB? status? / blast~~| ~~1h~~ | ~~jg/cp~~ | ~~to-be-prepared (show 18)~~
PrepareButSkip | Other programmatic tools for downstream analysis (BiomaRt, StringDB, Setrank) | Biomart, String, Setrank | 1h | cpNwew | to-be-prepared (show 21)
~~Skip~~ | ~~RNAseq vs LFQ, What is to expect~~ | ~~Number game, log2Correlation~~ | ~~1h~~ | ~~jg~~ | ~~Combo-course day5 (show 22)~~

### Day 4 -> moved to Day ZERO

- Socio Economics
-> Phone Call scheduled Thursday, 27th of Oct.
- email: ingrid.fromm@bfh.ch
- phone: 031 910 2192 
- LI: https://www.linkedin.com/in/ingrid-fromm-phd-8a74085b


Phone conference: Thu, 27.10.2016 (11.45 - 12:03)
- first get in touch: Week: 23.-27. January -> Tentatively fixed.. !!
- where do you see common ground? Where? -> aware of the problem?
--> somebody from ETH doing socio economics.. (Ralph?)
--> Overview of Socio Enomics .. 
- what do you do?
--> We get a simple presentation.. & I will send something similar.. 
- what kind of data do you have? --> we will find out from her presentations
- what kind of statistics do you do? --> we will find out from her presentations 
- After reading the presentation I will have a hard time to find common ground

Presentation content (I. Fromm):
- Regional aspects with respect to production over the past decades
- Which varieties are grown where? 
- Which aspects are important for growing this variety and not the other

### Common Ground: (potentially on day 1)
- How is your data "sampled"?
- What is the N?
- How is N defined?
- How are outliers identified and handled?
- How are missing values handled?
- What statistics are used to analyze the data?


## Organisational issues:
(check here for templates: https://lgatto.github.io/2014-11-06-UZH/)

### Student should bring:
- own laptop (Windows or Mac/Linux w Virtual Box and Windows instance)
- equiped with: Editor (vi, nano, ..), R., RStudio, git
- some free space (20gigs at least)


## Students
- 8-10 PhD students/postdocs
- some knowledge of biology (DNA -> mRNA -> Proteins -> Proteinfuntions (enzymes)
- proven skills in (ideally) R (or python)


## Publications to study (upfront)
- Review (Aebersold n Mann) -> https://www.ncbi.nlm.nih.gov/pubmed/27629641
- MQ -> https://www.ncbi.nlm.nih.gov/pubmed/24942700
- LFQ -> https://www.ncbi.nlm.nih.gov/pubmed/23391308
- MSQC1 -> https://www.ncbi.nlm.nih.gov/pubmed/27130639
- Top3 Protein quantification -> https://www.ncbi.nlm.nih.gov/pubmed/20576481


## Notes by CP

### add visualization techniques 15min from combined course 
- plus t-test alternative for boxplot N(0,1) distribution example. 

### References
- Visualizing data, William S. Cleveland, Murray Hill, New Jersey : AT&T Bell Laboratories 1993
- Lattice, Multivariate Data Visualization with R, Deepayan Sarkar , 2008
- Volcano Plot -> https://www.ncbi.nlm.nih.gov/pubmed/27272648


