# ISCB2017

## Draft Overview of the week

## Idea: Little modules (theory.. and respective hands-on session)

### Day 1

**Listing of modules (buzzword & idea for hands-on):**
- This week (outline, instructors, goals)
- FGCZ (Functional Genomics Center Zurich) a core facility for Zurich Lifesciences community
- Mass spectrometry & protein analytics 
- Protein identification w mass spectrometry & databases // Wrkshp: digest & theoretical considerations about search space and different DBs
- MS and different activation techniques // Wrkshp: protViz & labelling of spectra
- Validation of database results using target-decoy strategy // Wrkshp: filter lists and calculate FDRs on different levels 
- PTMs
- Experimental desgin (potential pitfalls) // Wrkshp: annotation in Bfabric
- Tools for protein identification // Wrkshp: get MQ installed and run it
- Conclusion day 1


### Day 2
- Repetition and wrap up of Day 1
- Quantitative proteomics strategies (an overview)
- Label-free quantification // Wrkshp: txt-tables of MQ
- Quantitative protein matrices // Wrkshp: Visualizing QMs
- Two group designs for Yeast, grown on different nutrient sources // Wrkshp: Analysing 2 groups
- Conclusion day 2


### Day 3
- Repetition and wrap up of Day 2
- Model organisms vs custom databases 
- Webtools for model organisms // Wrkshp: Visualizing results on a cellular overview map (YeastCyc)
- Principle of ORAs // Wrkshp: Webgestalt w YEAST results
- Other programmatic tools for downstream analysis (BiomaRt, StringDB, Setrank) // Wrkshp: Biomart, String, Setrank
- RNAseq vs LFQ, What is to expect // Wrkshp: Number game, log2Correlation
- Conclusion day 3

### Day 4
- Socio Economics
--> Fromm Ingrid, email: ingrid.fromm@bfh.ch, 031 910 2192 
--> https://www.linkedin.com/in/ingrid-fromm-phd-8a74085b
--> http://www.hafl.bfh.ch/?id=625&no_cache=1&tx_smemployeelist_pi1[empID]=329


## Organisational issues:
(check here for templates: https://lgatto.github.io/2014-11-06-UZH/)

### Student should bring:
- own laptop
- equiped with: Editor (vi, nano, ..), R., RStudio, git
- some free space (20gigs at least)


## Students
- 8-10 PhD students/postdocs
- some knowledge of biology
- proven skills in at least one of the following programming languages (R, python)


## Publications to study (upfront)
- Review (Aebersold n Mann) -> https://www.ncbi.nlm.nih.gov/pubmed/27629641
- MQ (Mann..)
- Weisser.. 