# ISCB2017

## Draft Overview of the week

## Idea: Little modules (theory.. and respective hands-on session)

**Listing of modules (buzzword & idea for hands-on):**



### Day 1

Theory Module | Hands-on Module | Duration | Responsible person
------------ | ------------- | ------------- | ------------- 
Outline for this week | break | 0.5 h | xxx
The FGCZ in Zurich | browse FGCZ page | 0.5 h | xxx
Mass spectrometry (and protein analytics) | browse around for milestones in MS, form groups, introduce yourself | 1 h | xxx
Getting ready for hands-on sessions | Install MQ, do git co, commits, get R packages | 2 h | xxx
Protein identification with search engines | get fasta, R (seqinR), R (digest), theoretical considerations about search space, different DBs, uniqueness of peptides | 2h | xxx
MS and different activiation techniques | protViz & mgfs & labelling of spectra | 1h | xxx
PTMs | ?? | 1h | xxx
Experimental design in proteomcis experiments | Annotation of samples in Bfabric | 1h | xxx
Conclusion of day 1 | Q & A | 1h | xxx



### Day 2

Theory Module | Hands-on Module | Duration | Responsible person
------------ | ------------- | ------------- | ------------- 
Repetition and wrap up of Day 1 | Q & A | 0.5 h | xxx
Tools for protein identification | Look at MQ and start a search | 1h | xxx
Quantitative proteomics strategies (an overview) | break | 1h | xxx
Label-free quantification | MQ: txt-tables | 1h | xxx
Quantitative protein matrices | Visualizing QMs in R. | 1h | xxx
Two group analysis for Yeast, grown on different nutrient sources | Analysing 2 groups in R. | 2h | xxx
Conclusion of day 2 | Q & A | 1h | xxx



### Day 3
Theory Module | Hands-on Module | Duration | Responsible person
------------ | ------------- | ------------- | ------------- 
Repetition and wrap up of Day 2| Q & A | 0.5 h | xxx
Model organisms vs custom databases | Pigeon pea DB? status? | 1h | xxx
Webtools for model organisms | Visualizing results on a cellular overview map (YeastCyc) | 1h | xxx
Principle of ORAs |  Webgestalt w YEAST results | 1h | xxx
Other programmatic tools for downstream analysis (BiomaRt, StringDB, Setrank) | Biomart, String, Setrank | 1h | xxx
RNAseq vs LFQ, What is to expect | Number game, log2Correlation | 1h | xxx
Conclusion of day 3 | Q & A | 1h | xxx



### Day 4

- Socio Economics
--> Fromm Ingrid, email: ingrid.fromm@bfh.ch, 031 910 2192 
--> https://www.linkedin.com/in/ingrid-fromm-phd-8a74085b
--> http://www.hafl.bfh.ch/?id=625&no_cache=1&tx_smemployeelist_pi1[empID]=329


## Organisational issues:
(check here for templates: https://lgatto.github.io/2014-11-06-UZH/)

### Student should bring:
- own laptop (Windows or Mac/Linux w Virtual Box and Windows instance)
- equiped with: Editor (vi, nano, ..), R., RStudio, git
- some free space (20gigs at least)


## Students
- 8-10 PhD students/postdocs
- some knowledge of biology
- proven skills in at least one of the following programming languages (R, python)


## Publications to study (upfront)
- Review (Aebersold n Mann) -> https://www.ncbi.nlm.nih.gov/pubmed/27629641
- MQ (Mann..)
- Weisser.. 